hyperspy/tests/io/test_hdf5.py

```diff
@@ -371,6 +371,7 @@ def teardown_method(self, method):
 
 class Test_permanent_markers_io:
 
+    @pytest.mark.skipif("sys.platform == 'darwin'")
```

Comment:
I would prefer xfail instead (http://doc.pytest.org/en/latest/skipping.html#mark-a-test-function-as-expected-to-fail), which would allow mac users to run full test suite locally, and would also notify us if Travis setup changes..